canonical-web-and-design/tutorials.ubuntu.com
# Added md file extension to onlyoffice server tutorial file

## Done

Added md file extension.

## QA

- Check out this feature branch
- Run the site using the command `./run serve`
- View the site locally in your web browser at: [http://0.0.0.0:8016/](http://0.0.0.0:8016/)
- Ensure the "How to Install ONLYOFFICE on Ubuntu 18.04" tutorial is available
